watershed_workflow/triangulation.py

```diff
@@ -1,6 +1,8 @@
 """Triangulates polygons"""
 import logging
 import collections
+from typing import List
```

Comment:
Is this actually being used?  I don't think we have typing information in actual code here?